03-specifying-sample-designs.Rmd

```diff
@@ -210,7 +212,7 @@ In the chunk above, the design object is printed, and the object summary is show
 
 - **Description**: A population is divided into mutually exclusive subpopulations (strata), and then samples are selected independently within each stratum.
 - **Requirements**: The sampling frame must include the information to divide the population into groups for every unit.
-- **Advantages**: This design ensures sample representation in all subpopulations. Often, for the same sample size as a SRS sample, the standard errors are smaller, so it is a more efficient design. This is true if the strata are correlated with outcomes.
+- **Advantages**: This design ensures sample representation in all subpopulations. Often, for the same sample size as a SRS sample, the standard errors are smaller, so it is a more efficient design. This is true if the strata are correlated with outcomes. 
 - **Disadvantages**: Auxiliary data may not exist to divide the sampling frame into groups, or the data may be outdated.
 - Examples: 
  - **Example 1**: A population of North Carolina residents could be separated into urban and rural areas, and then a SRS of residents from both rural and urban areas is selected independently. This ensures there are rural residents in the sample.
```

Comment:
At least in the HTML book, these bullets don't indent past the first indentation.

Reply:
Yes, they need to be indented once more. I think the linting may have messed with it.